x-pack/plugin/deprecation/qa/rest/src/javaRestTest/java/org/elasticsearch/xpack/deprecation/DeprecationHttpIT.java

```diff
@@ -376,7 +376,7 @@ public void testDeprecationMessagesCanBeIndexed() throws Exception {
                             hasEntry("ecs.version", "1.7"),
                             hasEntry(KEY_FIELD_NAME, "deprecated_settings"),
                             hasEntry("event.dataset", "deprecation.elasticsearch"),
-                            hasEntry("log.level", "DEPRECATION"),
+                            hasEntry("log.level", "CRITICAL"),
```

Comment:
It would be good if we could also trigger a WARN level deprecation and observe that it is written to the data stream.

Reply:
will add